tempesta_fw/http_sess.c

```diff
@@ -736,12 +753,16 @@ TfwCfgMod tfw_http_sess_cfg_mod = {
 			.allow_none = true,
 		},
 		{
+			/* Value is parsed as int, set max to INT_MAX*/
 			.name = "sess_lifetime",
 			.deflt = "0",
-			.handler = tfw_cfg_set_int,
+			.handler = tfw_http_sticky_sess_lifetime_cfg,
 			.dest = &tfw_cfg_sticky.sess_lifetime,
```

Comment:
In `tfw_http_sticky_sess_lifetime_cfg` i use `tfw_cfg_set_int` for parsing int value and check it for min-max borders. To use that i need to set `.dest` either here or in `tfw_http_sticky_sess_lifetime_cfg`.